source/vocab/details.ttl

```diff
@@ -1254,6 +1272,11 @@
     owl:propertyChainAxiom ( :hasDuration :value );
     owl:equivalentProperty bf2:duration .
 
+:extentLabel a owl:DatatypeProperty ;
```

Comment:
Maybe fits better as extentNote?

Reply:
Much better! fixed in efa01444a0b8e5f73321dfe5656b5fe643e78a02.